schemas/settings.json

```diff
@@ -0,0 +1,144 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "definitions": {
+        "color": {
+            "type": "string",
+            "enum": [
+                "red",
+                "green",
+                "yellow",
+                "blue",
+                "magenta",
+                "cyan",
+                "bold",
+                "underlined",
+                "red_bold",
+                "green_bold",
+                "yellow_bold",
+                "blue_bold",
+                "magenta_bold",
+                "cyan_bold",
+                "red_underlined",
+                "green_underlined",
+                "yellow_underlined",
+                "blue_underlined",
+                "magenta_underlined",
+                "cyan_underlined",
+                "~"
```

Comment:
An unquoted `~` means null in YAML - is it ok to define it like a string in the schema?

Reply:
Otherwise, this value will not be accepted while docs [list](https://bashly.dannyb.co/usage/settings/#usage_colors) it as the default valid one. So yeah, it's okay. From this perspective. Another question how extension being used inserts such string when suggesting it - in quotes or not. VS Code for instance put it unquoted what results in type error telling that string expected, but it's an extension issue. :)